CHANGES.txt

```diff
@@ -9,6 +9,7 @@ Changes
   DLLs will be loaded from .libs (for wheels) or from directories in PATH
   environment variable.
 - Add version to CRS.to_wkt() for WKT2 support (#2122).
+- WarpedVRT: Remove deprecated dst_* parameters (#2197).
```

Comment:
@idanmiara just a tiny nit: can you move your change note to the top of the stack? I'd suggest it inline but can't figure out how.

Reply:
Sure, thanks!